### ISSUE TITLE
Email notification with ticket-message when Message is private

### ISSUE THREAD

**Issue by Nikhos** [opensupports/opensupports#513]
Hey there!
I'm using version 4.4.0.
When created a private message in a ticket, the user is getting a mail with the content of my private message. When the user opens the ticket, the message is invisible.
Can you fix this?

**Comment by vidiroli** [comment]
Confirmed, plz fix it.
Many thx

**Comment by lyrics1988123** [comment]
Omg.. have this too, please fix asap - that is horrible!!!

**Comment by lyrics1988123** [comment]
possible a quickfix:
could we disable the preview of the ticketentry in the emailnotification, so you only get the title and the info, that something happend?

**Comment by mrvice** [comment]
Admin view
![Adminansicht_Ticket](https://user-images.githubusercontent.com/24787896/60813487-233ddf80-a194-11e9-9f87-a3fb11dfe9f4.JPG)

User receives mail < user can read privat area message
![Usermail privatgelände](https://user-images.githubusercontent.com/24787896/60813504-2e910b00-a194-11e9-8339-b105bb4d6036.JPG)

user opens ticket cannot see privat area message
![UserTicketaufruf](https://user-images.githubusercontent.com/24787896/60813521-3b156380-a194-11e9-9eeb-989f8f418a64.JPG)

definitely a very annoying bug private area messages are not usable atm.

**Comment by mredigonda** [comment]
Yes, this is a critical issue and we are working on it right now, it will be ready for the next release.
